src/main/java/org/kairosdb/core/http/rest/MetricsResource.java

```diff
@@ -339,34 +371,42 @@ public Response getMeta(String json)
 		}
 		catch (JsonSyntaxException e)
 		{
+			span.log(e.getMessage());
```

Comment:
Does it make sense to set an `ERROR` tag as well? It doesn't look like a "soft error" that can be reported as a log message.
If you decide to add `ERROR` tag, then it can be done like this:
```
Tags.ERROR.set(span, Boolean.TRUE);
```